backend/app/Models/Grade.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+
+class Grade extends Model
+{
+    //
+        public function student():BelongsTo
+    {
+        return $this->belongsTo(Student::class, 'student_id','id');
```

Comment:
@ndvkhangsgt 
You can write as this if the foreign key and primary key are standard of Laravel.
Document at [here](https://laravel.com/docs/9.x/eloquent-relationships#one-to-many).
```suggestion
        return $this->belongsTo(Student::class);
```